Add 2023-12-13 agenda and minutes (#604)
solid/specification: meetings/2023-12-13.md

* SC: 2023-12-05 [W3C Solid Community Group and Social Community Group Joint Meeting](https://github.com/solid/specification/blob/main/meetings/2023-12-05.md) went great / people stuck around for another ~45 minutes. We'll follow-up on specific topics to discuss.
* SC: 2023-12-12 [Authorization Service/Agent/App](https://github.com/solid/specification/discussions/603) went great. Some notes to follow.
* eP: Just put the notes in the discussion. They're a bit random, but relevant links if you attended. We discussed the Launcher App (Henry and Gordana). It's been brought up since 2019. We plan to follow up in January once we have more clarity. Michiel demonstrated another launcher experiment he worked on with Inrupt years ago. I mentioned SAI authorization agent, I plan to record more videos and possibly have another meeting Jan/Feb and follow up on details. Mostly focused on dedicated app handling access and independent efforts have emerging approach with a dedicated authz party as complementary/alternative to having every app fiddling with access. 

### We Distribute
* SC: https://wedistribute.org/ wants to cover news about Solid...

> * SC: If not specifically about CG / specifications, I suggest to let them coordinate with Solid Team.
> * LH: Started very long time ago with diaspora and transferred into the fediverse. Started with interest on Social Web, then AP and now we see more interest in open protocols. There's so much info out there and one of the struggles I find is it's hard to understand what's going on. For Solid it's about where it'd fit in so we're here to see what people are building. 

Reviewer: ```suggestion
* SC: If not specifically about CG / specifications, I suggest to let them coordinate with Solid Team.
* LH: Started very long time ago with diaspora and transferred into the fediverse. Started with interest on Social Web, then AP, and now we see more interest in open protocols. There's so much info out there, one of the struggles I find is it's hard to understand what's going on. For Solid, it's about where it'd fit in, so we're here to see what people are building. 
```

> * DO: In the last year with the rise of AP/fediverse in the media with all the issues around twitter and reddit, there's a lot of overlap and alignment. To me the general public is not aware the these protocols have been in existence for well over a decade. We're in a very unique time where these foundations... with newer protocols there's a lot happening in Social Web space, many different approaches, users can have freedom in terms of their data, their social graph and what that can look like... each protocol takes different approach, there can be some adoption and integration that can make more complete approaches as opposed to what has been done separately. we want to connect with everyone involved, make sure we can bring awareness to the projects. 

Reviewer: ```suggestion
* DO: In the last year, with the rise of AP/fediverse in the media, with all the issues around Twitter and Reddit, there's a lot of overlap and alignment. To me, the general public is not aware the these protocols have been in existence for well over a decade. We're in a very unique time where these foundations... with newer protocols, there's a lot happening in Social Web space; many different approaches; users can have freedom in terms of their data, their social graph, and what that can look like... each protocol takes a different approach. There can be some adoption and integration that can make more complete approaches, as opposed to what has been done separately. We want to connect with everyone involved, make sure we can bring awareness to the projects. 
```

> * eP: Thank you for the work you do. Often people who work on specs/implementing become very focused on that and sometimes not good outreach, so what you do is important. How big is the audience of wedistribute? And do you work with articles or other media (video, interviews)? 
> * LH: It's not big. We're the largest probably on the fediverse but ??? Mainly articles, there's work on a podcast, and interviews. Even developers don't talk to each so it's a way of getting to know other similar projects. 

Reviewer: ```suggestion
* eP: Thank you for the work you do. Often people who work on specs/implementing become very focused on that and sometimes lack good outreach, so what you do is important. How big is the audience of wedistribute? And do you work with articles or other media (video, interviews)?
* LH: It's not big. We're the largest probably on the fediverse but ??? Mainly articles, there's work on a podcast, and interviews. Even developers don't talk to each other so it's a way of getting to know other similar projects.
```

> * DO: No set number. We had an article last week about 1000 reads. Depends on the topic. We have accounts on decentralized and centralized platforms. Making an effort to bolster our social media presence. We have started a podcast that we haven't publicly lauched but we have 4-5 episodes with developers, co-authors of AP and some with AT protocol in the works.. we want to extend that to cover Solid as well. We have an informal relationship with Sebastien Rosset. We're also working on some video media utilising PeerTube platform. Publication has been around for some time. 

Reviewer: ```suggestion
* DO: No set number. We had an article last week get about 1000 reads. It depends on the topic. We have accounts on both decentralized and centralized platforms and are making an effort to bolster our social media presence. We have started a podcast that we haven't publicly launched, but we have 4-5 episodes with developers, co-authors of AP, and some with AT protocol in the works. We want to extend that to cover Solid as well. We have an informal relationship with Sebastien Rosset. We're also working on some video media utilising the PeerTube platform. Publication has been around for some time. 
```

* SC: FYI, there was some coverage of dweb/social approaches in https://gitlab.com/bluesky-community1/decentralized-ecosystem that you might find useful, if not already in your radar.
* SC: Happy to help if you'd like to go over Solid Protocol or other specs: https://solidproject.org/TR/

### WIP Implementation Feedback

* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.

> * eP: More of a question: been working on establishing mutual relationship between two social agents. Have a working flow based on SAI. Currently relies on SAI sending through a message (sms, email). I want to optimize copy/paste flow so started using [webshare API](https://www.w3.org/TR/web-share/). Wanted to ask if people have experience with [web share target](https://w3c.github.io/web-share-target/) because it's not consistence across platforms. 

Reviewer: ```suggestion
* eP: More of a question: been working on establishing mutual relationship between two social agents. Have a working flow based on SAI. Currently relies on SAI sending through a message (sms, email). I want to optimize copy/paste flow so started using [webshare API](https://www.w3.org/TR/web-share/). Wanted to ask if people have experience with [web share target](https://w3c.github.io/web-share-target/) because it's not consistent across platforms. 
```

* VB: May want to ask this in solid/app-development chat.

### Solid CG Chair Election Procedure
URL: https://github.com/solid/specification/discussions/582

* SC: Voting closed. Requested W3C Team (Community Process) to give us the anonymised ballot data that's to be computed: https://lists.w3.org/Archives/Public/public-solid/2023Dec/0016.html
* eP: W3C takes holiday break next week?
* SC: Yes, I think 18th. Officially. 

> * eP: If they find it inconvenient to make a PR and send you data instead that'd be okay with you making the PR and they just approve or comment on that PR. I don't want to delay because they don't feel comfortable. 
> * SC: They're aware of the procedure. If there were issues they'd have mentioned or would let us know or just send me the data anyway. I don't expect any slowdown. They give us the data, we compute, we give to them, they approve and announce it. 

Reviewer: ```suggestion
* eP: If they find it inconvenient to make a PR and send you data instead, that'd be okay, with you making the PR and they just approve or comment on that PR. I don't want to delay because they don't feel comfortable. 
* SC: They're aware of the procedure. If there were issues, they'd have mentioned or would let us know or just send me the data anyway. I don't expect any slowdown. They give us the data, we compute, we give to them, they approve and announce it. 
```

### i18n and n11n of resource identifiers
URL: https://github.com/solid/specification/pull/575

* SC: All, please review.

> * SC: We're considering using IRIs instead of URIs, having mapping around.. gets a bit complicated. There's a number of specs involved. I reviewed, would be good to have more reviews. I'll follow up on my part. I think WT did a great job detailing what implementers need to do, but something like this needs deeper reviews, not just thumbs up to challenge each detail because things can be horribly wrong when ??? There's a parallel discussion on WebID CG about this topic. However we decide here will also influence how we use WebIDs. We need to have some implementation feedback on this. WT did a good job detailing what should happen, but not sure to what extent existing implementations will conform, and we don't have tests or a way to verify it. Hesitant to throw something like this into the spec and hoping things go smoothly. There's an alternative approach not to introduce this. If we use internationalized strings, we need to detail the conversion as for example WT ??? It comes down to how much interest people have. 

Reviewer: ```suggestion
* SC: We're considering using IRIs instead of URIs, having mapping around.. gets a bit complicated. There's a number of specs involved. I reviewed, would be good to have more reviews. I'll follow up on my part. I think WT did a great job detailing what implementers need to do, but something like this needs deeper reviews, not just thumbs up, to challenge each detail because things can go horribly wrong when ??? There's a parallel discussion on WebID CG about this topic. However we decide here will also influence how we use WebIDs. We need to have some implementation feedback on this. WT did a good job detailing what should happen, but not sure to what extent existing implementations will conform, and we don't have tests or a way to verify it. Hesitant to throw something like this into the spec, hoping things go smoothly. There's an alternative approach not to introduce this. If we use internationalized strings, we need to detail the conversion as for example WT ??? It comes down to how much interest people have. 
```

Reviewer: This shows why long lines can be problematic. My changes (of which there are several, mostly tiny) are not properly highlighted inline.

> * eP: Is there any changes regarding normalization? I see mention of an issue.. not sure if it's addressing in focused way. 
> * SC: There's a couple of issues across different repos touching on this topic. What WT did was gathering things that are impacted by this. WAC is another case. Kjetil covered it broadly. 

Reviewer: ```suggestion
* eP: Are there any changes regarding normalization? I see mention of an issue.. not sure if it's being addressed in a focused way. 
* SC: There are a couple of issues across different repos touching on this topic. What WT did was gather things that are impacted by this. WAC is another case. Kjetil covered it broadly. 
```

* eP: I'll review and pick up again if needed.
* SC: Be rough on it. Sometimes we let some features go through with very loose/non-strict reviews. This is one that needs a strong review. Challenge it to the point of "we don't need it", because other communities have avoided this possibility. I plan to compare it with approach to IRIs in [RDF Dataset Canonicalization](https://www.w3.org/TR/rdf-canon/)

### Chair Election 2023 Retrospective

> * eP: For first meeting in January we can schedule retrospective on election process. Good opportunity for feedback/suggestions. 

Reviewer: ```suggestion
* eP: For first meeting in January, we can schedule a retrospective on election process. Good opportunity for feedback/suggestions. 
```

* eP: How about unlocking election procedure discussion and linking to that in case people have feedback?

> * SC: Kept it locked because I didn't want it to turn into a major discussion where people need to walk through threads and get into discussion. Going forward, I'd like to suggest we put this document into w3c-cg/solid repo and keep there, for future elections it can be copied.

Reviewer: ```suggestion
* SC: Kept it locked, because I didn't want it to turn into a major discussion where people need to walk through threads to get into the discussion. Going forward, I'd like to suggest we put this document into w3c-cg/solid repo and keep it there. For future elections, it can be copied.
```

* eP: +1
* VB: +1

> * SC: W3C Team Community Process is interested in using this for any other CG who are interested. We're the pioneers :) and they're quite happy with the process. Credentials CG are having their own elections.. they won't use our process but they might in the future.
> * eP: Do you have links to Credentials CG election? Would be useful to have.. might be interesting to invite them. If there's opportunity to collaborate maybe they'd like to join the meeting. 

Reviewer: ```suggestion
* SC: W3C Team Community Process is interested in using this for any other CG who are interested. We're the pioneers :) and they're quite happy with the process. Credentials CG are having their own elections. They won't use our process now, but they might in the future.
* eP: Do you have links to Credentials CG election? Would be useful to have... might be interesting to invite them. If there's opportunity to collaborate, maybe they'd like to join the meeting. 
```

* SC: It's in the mailing list. They have a charter and an announcement. Our charters are very close to each other, couple details are different as far as process. They don't detail the chair election procedure beyond what's in the charter, AFAIK. They have been doing elections for some years now. 
* eP: Do we agree we can copy discussion into CG repo and unlock to use as async preparation for retro? 

> * SC: I wanted to create a snapshot of the procedure that's used and keep as is. You want to include feedback on it, that's reasonable, we can unlock the discussion for that meeting and write notes the, but I do want a copy of that procedure in the W3C CG repo. 
> * eP: I can make a PR..
> * SC: It's still ongoing. Once I click buttons to assign new chairs it's all wrapped up.

Reviewer: ```suggestion
* SC: I wanted to create a snapshot of the procedure that's used, and keep it as is. You want to include feedback on it, that's reasonable, we can unlock the discussion for that meeting, and write notes there, but I do want a copy of that procedure in the W3C CG repo. 
* eP: I can make a PR.
* SC: It's still ongoing. Once I click buttons to assign new chairs, it's all wrapped up.
```
